packages/create-analog/index.js

```diff
@@ -24,6 +24,11 @@ const APPS = [
         display: 'TypeScript',
         color: green,
       },
+      {
+        name: 'angular-v17-blog',
```

Comment:
I'd just use `blog` here and assume you're using the latest version of Angular and Analog. And change the folder name to `template-blog`.

```suggestion
        name: 'blog',
```